projects/plugins/jetpack/wp-cli-templates/block-register-php.mustache

```diff
@@ -2,7 +2,7 @@
 /**
  * {{ title }} Block.
  *
- * @since 12.1
+ * @since $$next-version$$
```

Comment:
The problem is that when the next release happens this will be replaced like every other instance of `$$next-version$$`. You already tried this in #30079, for example, and it got replaced in #30127.

Can you do something like `{{ next-version-literal }}` here in the template and have the code in whatever is supplying the value for `{{ title }}` above also supply this literal string for that variable?

Reply:
heh, I had forgotten that I had already done that. Your idea sounds good. done in 0ca1f959a0b4b9d1c302f92d5787eb0d5c9db074